ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -109,6 +113,22 @@ export const Y_AXIS_CONFIG = {
   max: { label: 'Max' },
 };
 
+// TODO: add Percent mode support for stacked area charts
+export type StackOptions = 'None' | 'Normal';
```

Comment:
I found the "Normal" name kind of confusing, but I think it'll make more sense once the "Percent" is in there.

Reply:
Yeah that’s true, would changing to all “All”

Reply:
Ended up changing to use 'All' instead of 'Normal'